### PR TITLE
Remove proto copy and use the snapshot release.

### PR DESCRIPTION
**Pull request by bogdandrutu**
Need help a bit with gradle, I am not sure how to configure the repositories to download the proto snapshot from https://oss.sonatype.org/content/repositories/snapshots/io/opentelemetry/opentelemetry-proto/0.1.0-SNAPSHOT/

Please give me any advice.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,8 @@
 buildscript {
```
Comment (pavolloffay) [inline build.gradle]: Lines 74 and 75 can be also removed
```
        // Ignore warnings for protobuf generated files.
        it.options.compilerArgs += ["-XepExcludedPaths:.*/sdk/build/generated/source/proto/.*"]
```

**Comment by bogdandrutu** [comment]
@pavolloffay do you want to take this and use gitsubmodules as you suggested?

**Comment by pavolloffay** [comment]
@bogdandrutu I did an experiment with it. I have added proto as gitsubmodule and then configured our gradle to link it as gradle submodule https://github.com/open-telemetry/opentelemetry-proto. I have run into a couple of issues: 1. the proto gradle projects defines the same tasks (I had to remove them). 2. for some reason I run into this error which I could not resolve, even after I removed all proto dependencies and linked only proto project as submodule `- com.google.protobuf:protobuf-java between versions 3.7.0 and 3.6.1`


I can take this but I would go a slightly different direction. I would setup a gitsubmodule and create a gradle module which would copy the proto from git and build it.

**Comment by pavolloffay** [comment]
I have created a PR which pulls proto files from GIT submodule https://github.com/open-telemetry/opentelemetry-java/pull/504

**Pull request closed without merging**